core/data.go

```diff
@@ -625,3 +626,8 @@ type BlobVersionParameters struct {
 	MaxNumOperators uint32
 	NumChunks       uint32
 }
+
+func (ar *ReservedPayment) IsActive() bool {
+	now := uint64(time.Now().Unix())
+	return ar.StartTimestamp <= now && ar.EndTimestamp >= now
```

Comment:
are all `ReservedPayment`s guaranteed to have these timestamps populated?

Reply:
I think yes, because before we make ReservedPayments, they go through the convert function [here](https://github.com/Layr-Labs/eigenda/blob/96215b135230ca5310c3ee7ffcf73491638d8dee/core/eth/utils.go#L132)

Reply:
If reservation has zero timestamps and non-zero values for other fields, looks like it would still get converted successfully

Reply:
yes, that is only for valid reservations, but this check should handle checking if the reservation is active or not

Reply:
I don't think I understand what are your concern here 🤔 ?

Reply:
My concern is whether it's possible to have a valid reservation that's missing `EndTimestamp` for example.

Reply:
I guess the current logic would ignore those reservations if they're missing any timestamps

Reply:
Yep!